frontend/src/components/StatsPage.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Container, Button, Table } from "reactstrap";
+import { PlayerStats } from "../models/stats.model";
+import { RootSchema } from "../store/root-schema";
+import { Dispatch } from "../store";
+import { changeView } from "../store/actions";
+import { connect } from "react-redux";
+
+interface Props {
+  username: string;
+  stats: PlayerStats;
```

Comment:
shouldn't this be `stats?: PlayerStats`? You later check for if stats is defined